docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Teleport to create host users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+The following utilities should be available in the PATH:
+- useradd
+- userdel
+- usermod
+- groupadd
+- getent
+- visudo
+
+## Role configuration
+
+First, a role must be created with `create_host_user: true` in order to allow users
+in this role to have host users created for them.
+
+Example:
+```yaml
+kind: role
+version: v5
+metadata:
+  name: auto-user-groups
+spec:
+  options:
+    # allow automatic creation of users.
+    create_host_user: true
+  allow:
+    # username from external attribute.
```

Comment:
It might be clearer to use a static list of logins rather than an external trait here so we don't need to explain the use of the IdP-provided template variable.